ios/browser/api/ads/brave_ads.mm

```diff
@@ -1175,7 +1175,8 @@ - (void)getScheduledCaptcha:(const std::string&)payment_id
 }
 
 - (void)showScheduledCaptchaNotification:(const std::string&)payment_id
-                               captchaId:(const std::string&)captcha_id {
+                               captchaId:(const std::string&)captcha_id
```

Comment:
Should we DCHECK if these are empty strings or do we do this in the adaptive captcha code?

Reply:
@tmancey i am not sure how it's implemented in IOS. @kylehickinson